src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -98,6 +98,7 @@ export class GcpCreateServerApp extends LitElement {
   private project: Project;
   private billingAccounts: BillingAccount[] = [];
   private regionPicker: OutlineRegionPicker;
+  private billingAccountsRefreshLoop: NodeJS.Timeout = null;
```

Comment:
This is web code, not Node code.  You can use Number for this.  If tsc complains using `window.setTimeout` should fix the problem.

Reply:
Thanks, done.